charts/latest-v2/azuredisk-csi-driver/templates/csi-azuredisk-controller.yaml

```diff
@@ -207,6 +207,7 @@ spec:
           args:
             - "--v={{ .Values.controller.logLevel }}"
             - "--config=/etc/{{ .Values.controller.name }}/config.yaml"
+            - "--enable-mount-replicas={{ .Values.controller.enableMountReplicas }}"
```

Comment:
We should add `enable-mount-replicas` to the configMap defined before the deployment here instead of adding a command line parameter.